src/scripts/script.js

```diff
@@ -38,179 +42,200 @@ const getCurrentAdditionalTemplate = () => {
-        todoLINode.setAttribute("style", `background-color:${!completedStatus ? setTaskBackgroundColor(priority) : setTaskBackgroundColor()}`);
-        todoLINode.setAttribute("id", `${id} `);
-        currentNode.append(todoLINode);
-        todoLINode.innerHTML = getTasksLITemplate(title, text, completedStatus);
-
-    })
-    if (!completedStatus) {
-        const map_removeButton = document.querySelectorAll(".btn-danger");
-        setClicksOnNode(map_removeButton, removeToDoItem);
-
-        const map_editButton = document.querySelectorAll(".btn-info");
-        setClicksOnNode(map_editButton, editTodoTask);
-
-        const map_completeButton = document.querySelectorAll(".btn-success");
-        setClicksOnNode(map_completeButton, completeTodoTask);
-    }
-    getCountTodos();
-}
+  const currentTodos = getMapTodoFromLocalStarage().filter(
+    (todo) => todo.isCompleted == completedStatus
+  );
+  const currentNode = completedStatus
+    ? getElementById("completedTasks")
+    : getElementById("currentTasks");
+
+  currentTodos.map(({ id, title, text, radio: priority }) => {
+    const todoLINode = document.createElement("li");
+    todoLINode.setAttribute(
+      "class",
+      `list-group-item d-flex w-100 mb-2 ${
+        !completedStatus ? "currentTodo" : ""
+      }`
+    );
+    todoLINode.setAttribute(
+      "style",
+      `background-color:${
+        !completedStatus
+          ? setTaskBackgroundColor(priority)
+          : setTaskBackgroundColor()
+      }`
+    );
+    todoLINode.setAttribute("id", `${id} `);
+    currentNode.append(todoLINode);
+    todoLINode.innerHTML = getTasksLITemplate(title, text, completedStatus);
+  });
+  if (!completedStatus) {
+    const map_removeButton = document.querySelectorAll(".btn-danger");
+    setClicksOnNode(map_removeButton, removeToDoItem);
+
+    const map_editButton = document.querySelectorAll(".btn-info");
+    setClicksOnNode(map_editButton, editTodoTask);
+
+    const map_completeButton = document.querySelectorAll(".btn-success");
+    setClicksOnNode(map_completeButton, completeTodoTask);
+  }
+  setCountTodos();
+};
 
 const addTask = () => {
-    const btn_saveTask = getElementById("form-group-save-task");
-    $('#exampleModal').modal("show");
-    btn_saveTask.addEventListener("click", () => {
-        const id_dateValue = +new Date;
-        addTodoToLocalStorage(id_dateValue, id_dateValue, ...getModalValues(), false);
+  const btn_saveTask = getElementById("form-group-save-task");
+  $("#exampleModal").modal("show");
+  btn_saveTask.addEventListener("click", () => {
+    const id_dateValue = +new Date();
+    addTodoToLocalStorage(
+      id_dateValue,
+      id_dateValue,
+      ...getModalValues(),
+      false
+    );
+  });
+};
+function toggleStyle() {
+  const fontElement = document.querySelectorAll("h3");
+  if (this.checked) {
+    document.getElementsByTagName("body")[0].style.backgroundColor = "#000";
```

Comment:
create variable for constants. what you do here is a `magic code`. 
1) https://stackoverflow.com/questions/41725725/access-css-variable-from-javascript - you can use this approach
2) u can create variables like `--main-bg-color = "#000"` and then call it in code 
3) also you can change body styles like this `document.body.style.background = --main-bg-color`
https://stackoverflow.com/questions/197748/how-do-i-change-the-background-color-with-javascript

Reply:
Good option for future, i've fixed that